crates/vm/levm/src/vm.rs

```diff
@@ -147,6 +148,11 @@ impl VM {
             default_touched_storage_slots.insert(address, warm_slots);
         }
 
+        // Add precompiled contracts addresses to cache
+        for i in 1..10 {
+            default_touched_accounts.insert(Address::from_low_u64_be(i));
+        }
+
```

Comment:
Can't we iterate the constant we have in `precompiles.rs` with the list of precompile addresses?

Reply:
Added this as a to-do comment in [Add todo coment](https://github.com/lambdaclass/ethrex/pull/1473/commits/02226e44f79ba8450a4a93da995c0cb4fc743714)